docs/_sidebar.md

```diff
@@ -0,0 +1,11 @@
+-   Guides
+    -   [Udaru Overview](overview.md)
+    -   [Authorization Introduction](authorization-introduction.md)
```

Comment:
This can be removed. This file has been replaced by `overview.md` and I forgot to remove it in my last PR. Note PR to remove it here: https://github.com/nearform/udaru/pull/405